lib/e164/lookuptable.rb

```diff
@@ -1,646 +1,690 @@
+    '1203' => ['US', 'United States'],
+    '1205' => ['US', 'United States'],
+    '1206' => ['US', 'United States'],
+    '1207' => ['US', 'United States'],
+    '1208' => ['US', 'United States'],
+    '1209' => ['US', 'United States'],
+    '1210' => ['US', 'United States'],
+    '1212' => ['US', 'United States'],
+    '1213' => ['US', 'United States'],
+    '1214' => ['US', 'United States'],
+    '1215' => ['US', 'United States'],
+    '1216' => ['US', 'United States'],
+    '1217' => ['US', 'United States'],
+    '1218' => ['US', 'United States'],
+    '1219' => ['US', 'United States'],
+    '1220' => ['US', 'United States'],
+    '1223' => ['US', 'United States'],
+    '1224' => ['US', 'United States'],
+    '1225' => ['US', 'United States'],
+    '1227' => ['US', 'United States'],
+    '1228' => ['US', 'United States'],
+    '1229' => ['US', 'United States'],
+    '1231' => ['US', 'United States'],
+    '1234' => ['US', 'United States'],
+    '1239' => ['US', 'United States'],
+    '1240' => ['US', 'United States'],
+    '1248' => ['US', 'United States'],
+    '1251' => ['US', 'United States'],
+    '1252' => ['US', 'United States'],
+    '1253' => ['US', 'United States'],
+    '1254' => ['US', 'United States'],
+    '1256' => ['US', 'United States'],
+    '1260' => ['US', 'United States'],
+    '1262' => ['US', 'United States'],
+    '1267' => ['US', 'United States'],
+    '1269' => ['US', 'United States'],
+    '1270' => ['US', 'United States'],
+    '1272' => ['US', 'United States'],
+    '1274' => ['US', 'United States'],
+    '1276' => ['US', 'United States'],
+    '1279' => ['US', 'United States'],
+    '1281' => ['US', 'United States'],
+    '1283' => ['US', 'United States'],
+    '1301' => ['US', 'United States'],
+    '1302' => ['US', 'United States'],
+    '1303' => ['US', 'United States'],
+    '1304' => ['US', 'United States'],
+    '1305' => ['US', 'United States'],
+    '1307' => ['US', 'United States'],
+    '1308' => ['US', 'United States'],
+    '1309' => ['US', 'United States'],
+    '1310' => ['US', 'United States'],
+    '1312' => ['US', 'United States'],
+    '1313' => ['US', 'United States'],
+    '1314' => ['US', 'United States'],
+    '1315' => ['US', 'United States'],
+    '1316' => ['US', 'United States'],
+    '1317' => ['US', 'United States'],
+    '1318' => ['US', 'United States'],
+    '1319' => ['US', 'United States'],
+    '1320' => ['US', 'United States'],
+    '1321' => ['US', 'United States'],
+    '1323' => ['US', 'United States'],
+    '1325' => ['US', 'United States'],
+    '1326' => ['US', 'United States'],
+    '1327' => ['US', 'United States'],
+    '1330' => ['US', 'United States'],
+    '1331' => ['US', 'United States'],
+    '1332' => ['US', 'United States'],
+    '1334' => ['US', 'United States'],
+    '1336' => ['US', 'United States'],
+    '1337' => ['US', 'United States'],
+    '1339' => ['US', 'United States'],
+    # '1340' => ['US', 'United States'], us virgin islands
```

Comment:
All these commented ones had responses in the old format, e.g. this was `["VI", "Virgin Islands, US"]`. Should we just keep that the same?

Reply:
The old format one's are still there just down below.  This is entirely for keeping future diff's more sane.